### ISSUE TITLE
Hello, may I ask if the data in the test part of the data set partitioning is fixed? How did  choose it?

### ISSUE THREAD

**Issue by zc-maker** [gist-ailab/SleePyCo#8]
test_dataset = EEGDataLoader(self.cfg, self.fold, set='test')
Hello, may I ask if the data in the test part of the data set partitioning is fixed? How did  choose it?

**Comment by SeongjuLee** [comment]
Yes, we used fixed partitioning, as provided in split_idx.
For Sleep-EDF-2013 and Sleep-EDF-2018, we used the same split indices with TinySleepNet.
For other datasets, we divide the subjects in k-fold CV manners.
For example, with given subjects [0, 1, 2, 3, 4, 5] and 3-fold CV manners, the followings are examples of split:
```
[fold 1] test subjects : [0, 1], remaining subjects are randomly split into train/val as described in the paper
[fold 2] test subjects : [2, 3], remaining subjects are randomly split into train/val as described in the paper
[fold 3] test subjects : [4, 5], remaining subjects are randomly split into train/val as described in the paper
```
For random split, we used numpy functions with fixed random seed 42.
